contracts/DacToken.sol

```diff
@@ -1,9 +1,11 @@
 pragma solidity ^0.4.8;
 import "./zeppelin/token/StandardToken.sol";
 import './ITransferRegulator.sol';
+import "./DacHub.sol";
+import './DacHubClient.sol';
 
 // The DAC Token is a standard ERC20 token with 21 million units and 18 decimal places.
-contract DacToken is StandardToken {
+contract DacToken is StandardToken, DacHubClient {

```

Comment:
The token contract will use the hub to get the address of the contract that allows/disallows transfers.